configs/lip-a2/microscope.j5

```diff
@@ -41,83 +85,114 @@
                 "G54",
                 */
                 //More proper homed scheme
-                "G10 L2 P1 X-297.000 Y-197.000 Z-3.000",
+                //"G10 L2 P1 X-297.000 Y-197.000 Z-3.000",
+                //8/5 adjustment
+                //"G10 L2 P1 X-475.200 Y-315.200 Z-4.800",
+                //gearbox + 0.9 motor
+                "G10 L2 P1 X-475.200 Y-315.200 Z-48.000",

```

Comment:
Z should be -4.8 as it doesn't have the gearbox

Reply:
done